src/Servers/HttpSys/src/AuthenticationManager.cs

```diff
@@ -61,6 +61,11 @@ public bool AllowAnonymous
         /// </summary>
         public bool AutomaticAuthentication { get; set; } = true;
 
+        /// <summary> 
+        /// Sets the display name shown to users on login pages. The default is null. 
+        /// </summary> 
+        public string AuthenticationDisplayName { get; set; }
```

Comment:
Reference assemblies need updating.
https://github.com/dotnet/aspnetcore/blob/master/docs/ReferenceAssemblies.md

Reply:
I couldn't generate reference assembly with scripts, so I just edit the file manually. I hope it works.

Reply:
What happened when you tried?